python/ray/ml/predictor.py

```diff
@@ -1,13 +1,14 @@
 import abc
-from typing import Union, TYPE_CHECKING
 
 from ray.ml.checkpoint import Checkpoint
+from ray.ml.utils.conversion_utils import (
+    convert_data_batch_to_arrow,
+    convert_arrow_to_data_batch,
+    ArrowDataType,
+    DataBatchType,
```

Comment:
Had to move `DataBatchType` to `conversion_utils` to avoid a circular import. Will fix later